src/pybind/mgr/orchestrator/module.py

```diff
@@ -372,7 +372,7 @@ def _get_hosts(self, format: Format = Format.plain) -> HandleCommandResult:
             table.right_padding_width = 2
             for host in sorted(hosts, key=lambda h: h.hostname):
                 table.add_row((host.hostname, host.addr, ' '.join(
-                    host.labels), host.status.capitalize()))
+                    host.labels), host.status.capitalize() if host.status else 'OK'))
```

Comment:
Appears status can be `offline`, `maintenance` or empty str.... would the term `online` or `available` fit better?  

I wish we had an enum here instead of an arbitrary list of strings. :(

Reply:
I like ONLINE a lot better than OK. I'll change to that.

Reply:
Neither `maintenance` not `offline` is capitalized. What about lowercase `online´?